src/NServiceBus.Core.Tests/DelayedDelivery/RouteDeferredMessageToTimeoutManagerBehaviorTests.cs

```diff
@@ -18,12 +18,17 @@ class RouteDeferredMessageToTimeoutManagerBehaviorTests
         {
             var behavior = new RouteDeferredMessageToTimeoutManagerBehavior("tm");
             var delay = TimeSpan.FromDays(1);
-            var message = new OutgoingMessage("id", new Dictionary<string, string>(), new byte[0]);
 
             var headers = new Dictionary<string, string>();
             string destination = null;
-            var context = new RoutingContext(message, new UnicastRoutingStrategy("target"), null);
-            context.AddDeliveryConstraint(new DelayDeliveryWith(delay));
+            var context = new TestableRoutingContext
+            {
+                RoutingStrategies = new List<RoutingStrategy>
```

Comment:
`RouteDeferredMessageToTimeoutManagerBehavior` seems to be the only place using that setter to replace the whole collection.

But that seems to be a bit out of scope?